catalogue_pipeline/ingestor/src/test/scala/uk/ac/wellcome/platform/ingestor/IngestorFeatureTest.scala

```diff
@@ -103,29 +107,56 @@ class IngestorFeatureTest
         queue.url,
         invalidMessage
       )
+      withLocalElasticsearchIndex(itemType = itemType) { indexNameV1 =>
+        withLocalElasticsearchIndex(itemType = itemType) { indexNameV2 =>
+          val flags = sqsLocalFlags(queue) ++ esLocalFlags(
+            indexNameV1,
+            indexNameV2,
+            itemType)
+
+          withServer(flags) { _ =>
+            // After a message is read, it stays invisible for 1 second and then it gets sent again.
+            // So we wait for longer than the visibility timeout and then we assert that it has become
+            // invisible again, which means that the ingestor picked it up again,
+            // and so it wasn't deleted as part of the first run.
+            // TODO Write this test using dead letter queues once https://github.com/adamw/elasticmq/issues/69 is closed
+            Thread.sleep(2000)
 
-      val flags = sqsLocalFlags(queue) ++ esLocalFlags(indexName, itemType)
-
-      withServer(flags) { _ =>
-        // After a message is read, it stays invisible for 1 second and then it gets sent again.
-        // So we wait for longer than the visibility timeout and then we assert that it has become
-        // invisible again, which means that the ingestor picked it up again,
-        // and so it wasn't deleted as part of the first run.
-        // TODO Write this test using dead letter queues once https://github.com/adamw/elasticmq/issues/69 is closed
-        Thread.sleep(2000)
-
-        eventually {
-          sqsClient
-            .getQueueAttributes(
-              queue.url,
-              List("ApproximateNumberOfMessagesNotVisible")
-            )
-            .getAttributes
-            .get(
-              "ApproximateNumberOfMessagesNotVisible"
-            ) shouldBe "1"
+            eventually {
+              sqsClient
+                .getQueueAttributes(
+                  queue.url,
+                  List("ApproximateNumberOfMessagesNotVisible")
+                )
+                .getAttributes
+                .get(
+                  "ApproximateNumberOfMessagesNotVisible"
+                ) shouldBe "1"
+            }
+          }
         }
       }
     }
   }
+
+  private def sendToSqs(work: IdentifiedWork, queue: Queue) =
+    sqsClient.sendMessage(
+      queue.url,
+      JsonUtil
+        .toJson(
```

Comment:
minor: in most places we do:

```scala
import uk.ac.wellcome.utils.JsonUtil._
```

and use `toJson` directly. Might be worth doing here to reduce the size of this method.